docs/source/prometheus.rst

```diff
@@ -26,9 +26,11 @@ dask_scheduler_clients
     Number of clients connected
 dask_scheduler_desired_workers
     Number of workers scheduler needs for task graph
-dask_scheduler_gil_contention_total
-    Value representing cumulative total of GIL contention,
-    in the form of summed percentages.
+dask_scheduler_gil_contention_seconds_total
+    Value representing cumulative total of *potential* GIL contention,
+    in the form of cumulative seconds during which a thread held the GIL locked.
```

Comment:
```suggestion
    in the form of cumulative seconds during which any thread held the GIL locked.
```

nit but this feels slightly clearer to me ("a thread" initially made me think it was for some specific individual thread, and then I was confused)